label_studio/core/views.py

```diff
@@ -78,10 +87,16 @@ def version_page(request):
 
 def health(request):
     """ System health info """
-    logger.debug('Got /health request.')
-    return HttpResponse(json.dumps({
-        "status": "UP"
-    }))
+    redis = bool_from_request(request.GET, "redis", False)
+    if not redis:
+        logger.debug('Got /health request.')
+        return HttpResponse(json.dumps({
+            "status": "UP"
+        }))
+    else:
+        health_state = rqworker_healthcheck()
+        result = {"message": health_state[1]}
+        return JsonResponse(result, status=200 if health_state[0] else 500)
```

Comment:
we need to return "status": "UP" as it was before, otherwise we will break a lot of our client's setups.

Reply:
Fixed